.clang-format

```diff
@@ -0,0 +1,2 @@
+BasedOnStyle: Chromium
+ColumnLimit: 160
```

Comment:
Hmm.. why not have new/changes code formatted to 80 cols like most other projects?

From a cursory look at the binaryen code it looks like most of the long lines are comments.  I'm particularly keen on the nicely wrapping comments, so I would love to see this line removed :)    But perhaps we can land it like this and remove in the followup CL?

Reply:
I'd like to remove this too. If no one else has concerns. I'll remove it.